pyworkflow/project/project.py

```diff
@@ -703,8 +701,9 @@ def _updateProtocol(self, protocol, tries=0, checkPid=False,
                     protocol._outputs.append(attr)
 
             # Restore backup values
-            if protocol.useQueueForProtocol():
+            if protocol.useQueueForProtocol() and jobId:  # If jobId not empty then restore value as the db is empty
```

Comment:
Case for direct protocol launch from the GUI. Without passing through an scheduling process.

In this case the jobid is obtained by the GUI and the job id should be preserved.

In case of scheduling a protocol, the jobid  is obtained during the "scheduling job" and therefore is it written in the rub.db and should be taken.